modules/core/src/main/java/org/apache/ignite/spi/discovery/tcp/ServerImpl.java

```diff
@@ -7287,10 +7287,18 @@ private InetSocketAddress checkConnection(List<InetSocketAddress> addrs, int tim
                                     sock.connect(addr, perAddrTimeout);
 
                                     liveAddrHolder.compareAndSet(null, addr);
+
```

Comment:
Why we log here? We do not connect to a node. Just checking whether it is alive. We duplicate fuhrter message:
` log.info("Connection check to previous node done: [liveAddr=" + liveAddr + ", previousNode=" + U.toShortString(previous) + ", addressesToCheck=" + nodeAddrs + ", connectingNodeId=" + nodeId + ']')`

Reply:
You're right! There is no need for such excessive logging. I removed it